src/model/transaction/TransferTransaction.ts

```diff
@@ -315,4 +315,19 @@ export class TransferTransaction extends Transaction {
             ),
         });
     }
+
+    /**
+     * @internal
+     * Check a given address should be notified in websocket channels
+     * @param address address to be notified
+     * @param alias address alias (names)
+     * @returns {boolean}
+     */
+    public NotifyAccount(address: Address, alias: NamespaceId[]): boolean {
```

Comment:
applies to all ``NotifyAccount`` functions:

- If it's tagged as @internal, shouldn't it be a private method?
- I would expect NotifyAccount to be notifyAccount (camelCase)

Reply:
it should be lowe case

Reply:
Internal should be fine, Internal won't work on abstract. will update the method name 👍